lib/api_request.js

```diff
@@ -131,9 +131,10 @@ module.exports = function api_request (params, callback) {
       AwsHelper.log.info({'message': 'No Hotel Ids Supplied in SNS Message'});
       var options = make_options(params);
       http_request(options, function (err, data) {
-        if (err) {
-          AwsHelper.log.info({err: err}, 'No Hotels Package results');
-          return callback(err);
+        if (err || !data.result || !Array.isArray(data.result) || data.result.length === 0) {
```

Comment:
Yup :) it's a mirror of https://github.com/numo-labs/lambda-ne-classic-package-provider/blob/567e65f16d58769e5c7d72aca8c1133fbe0ea518/lib/api_request.js#L97 - should maybe throw in an `isArray()` check in there as well.